bluetooth_mesh/messages/test/test_time.py

```diff
@@ -0,0 +1,143 @@
+#
+# python-bluetooth-mesh - Bluetooth Mesh for Python
+#
+# Copyright (C) 2019  SILVAIR sp. z o.o.
+#
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+#
+import pytest
+
+from bluetooth_mesh.messages.time import TimeOpcode, TimeMessage, TimeRoleEnum
+
+valid = [
+    # fmt: off
+    pytest.param(
+        b'\x82\x37',
+        TimeOpcode.TIME_GET,
+        dict(),
+        id="TIME_GET"
+    ),
+    pytest.param(
+        b'\x5C\xbd\xf7\x67\x26\x00\x1a\xb2\x49\x02\x48',
+        TimeOpcode.TIME_SET,
+        dict(
+            tai_seconds=0x2667f7bd,
+            subsecond=0x1A,
+            uncertainty=0xB2,
+            time_authority=1,
+            tai_utc_delta=0b000000100100100,
+            time_zone_offset=0x48
```

Comment:
why hex?

Reply:
Same as above + both of them are signed asymmetrically, so I can't represent it easily as a integer. Maybe converting functions should be used instead of raw value.

Reply:
Yup!

Reply:
I found the best way - I will create new Construct field (like SensorData in this repo) where we will pass datetime that will set tai_seconds, subsecond tai_utc_delta and time_zone_offset field automatically ;). Also if we get a message, this Construct field will parse message and will return datetime field